bower.json

```diff
@@ -35,9 +35,9 @@
   ],
   "dependencies": {
     "firebase": "2.1.x",
-    "ember": "1.9.x",
     "ember-data": "1.0.0-beta.12"
   },
   "devDependencies": {
+    "ember": "1.9.x"
```

Comment:
you could be right. the `ember-data` dep is >=1.7.0 .. i wonder if that will install 1.10.0, which i have not tested...

I can test this now